packages/web/app/src/lib/preflight/shared-types.ts

```
@@ -2,6 +2,14 @@
 
 import { Kit } from '../kit';
 
+// todo stop using enums
+export const enum PreflightWorkerState {
+  running,
+  ready,
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)